### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by SiansandaTamara**
### Redux###
- No linter errors
- Use of reducers
- Use of actions
- Use of redux

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,24 +1,14 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import Books from './Books';
+// import PropTypes from 'prop-types';
+// import Books from './Books';
```
Comment (brytebee) [inline src/components/Book.js]: - [ ] **Optional:** To maintain a clean and readable codebase, kindly remove all commented and unused codes.

File: src/components/Book.js [hunk 2]
```diff
@@ -1,24 +1,14 @@
...
 
-const BookContent = ({ books }) => (
+const BookContent = () => (
   <ul className="book-content">
     {
-books.map((book) => <Books key={book.id} book={book} />)
+// books.map((book) => <Books key={book.id} book={book} />)
```
Comment (brytebee) [inline src/components/Book.js]: - [ ] **Optional:** Same as above

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @SiansandaTamara👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- `react-redux` is present in package.json.✔️
- The structure of the files follows the `ducks pattern`.✔️
- Defined `ADD_BOOK` and `REMOVE_BOOK` action types using the `ducks pattern`.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️

- [ ] There should be a `reducer` for the `categories` component which returns "Under Construction" when the button `check status` is clicked.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
Hi @SiansandaTamara,

Thank you for implementing changes requested by previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 
appr2


# Status: Approved ✔️✔️✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- For next project make sure that you used correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).In `gitflow`, To create pull request you compare `feature` branch with`development` branch.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**